### DISCUSSION TITLE
Does KaTeX Support Unicode?

### DISCUSSION THREAD

**Discussion by abdyek** [KaTeX/KaTeX#3901] [Q&A]
Hello everyone,

My native language is Turkish. KaTeX shows it turkish chars as different.
![image](https://github.com/KaTeX/KaTeX/assets/16313808/b4857765-0add-4999-81bc-55ed3fbf007b)
![image](https://github.com/KaTeX/KaTeX/assets/16313808/5903fdb4-2ebb-4996-9d37-72f93a7ac870)
Is there any way to fix it?

Thank you to read!

**Comment by pzinn** [comment]
can you post the actual code rather than a picture, so others can test it.

**Reply by abdyek** [reply]
Oh.. sorry

The actual  code:
```
[latex]$V_{çocuk}$[/latex]
```

**Reply by pzinn** [reply]
ah yes, the c with a cedilla under it is incorrected rendered. look like a true bug.

**Reply by pzinn** [reply]
you'll note that this works: `[latex]$V_{\text{çocuk}}$[/latex]`. note the cedillas do not exist in LaTeX in math mode. that being said, KaTeX shouldn't try to substitute `\c c`, so it's still a bug.

**Reply by pzinn** [reply]
please create an issue and I'll look into it.

**Reply by abdyek** [reply]
@pzinn Thank you. Do you mean that it is a bug? We can use it \text{all turkish chars: öçşığüÖÇŞİĞÜ} to show turkish chars but we prefer to use it without any prefix (and suffix) keywords if it is possible. :smile:

**Comment by pzinn** [comment]
yes it's a bug.
